scripts/generate_client_publish_job.py

```diff
@@ -0,0 +1,94 @@
+import argparse
+import os, subprocess
+import yaml
+
+WORKSPACE = os.environ.get("WORKSPACE")
+if WORKSPACE is None:
+    raise RuntimeError("provide the WORKSPACE variable!")
+
+
+def generate(trigger_from_repo, version_to_publish="master"):
+    release_tool = os.path.join(WORKSPACE, "integration", "extra", "release_tool.py")
+    integration_versions = subprocess.run(
+        [
+            release_tool,
+            "--integration-versions-including",
+            trigger_from_repo,
+            "--version",
+            version_to_publish,
+        ],
+        capture_output=True,
+    )
+
+    stage_name = "trigger"
+    document = {
+        "stages": [stage_name],
+        "trigger:mender-qa": {
+            "stage": stage_name,
+            # "rules": [{"if": "$CI_COMMIT_BRANCH == /^(master|[0-9]+\.[0-9]+\.x)$/"}],
```

Comment:
Shouldn't these rules be only in the caller's pipeline? I mean that `generate-qa-trigger` job in `mender-connect`, for example, already contains the branch limitation rules so I don't think they are needed here too.

Reply:
argh right, I forgot to actually remove them... 👌 should now be gone!